src/main/scala/org/hammerlab/guacamole/Concordance.scala

```diff
@@ -85,7 +86,8 @@ object Concordance {
     val filteredTrueGenotypes = trueGenotypes.filter(relevantVariants)
 
     val sampleName = filteredCalledAlleles.take(1)(0).getSampleId.toString
-    val sampleAccuracy = filteredCalledAlleles.concordanceWith(filteredTrueGenotypes).collectAsMap()(sampleName)
+
+    val sampleAccuracy = new GenotypeConcordanceRDDFunctions(filteredCalledAlleles).concordanceWith(filteredTrueGenotypes).collectAsMap()(sampleName)
```

Comment:
@ryan-williams I assume there's a more concise way to express this but I had trouble finding it.